src/main/resources/assets/tfc/models/item/metal/shield/bismuth_bronze.json

```diff
@@ -1,7 +1,409 @@
 {
-  "__comment__": "This file was automatically created by mcresources",
-  "parent": "item/handheld",
+  "credit": "Made with Blockbench",
+  "gui_light": "front",
   "textures": {
-    "layer0": "tfc:item/metal/shield/bismuth_bronze_front"
-  }
+    "1": "tfc:item/metal/shield/bismuth_bronze_front",
+    "2": "tfc:item/metal/shield/bismuth_bronze_back",
+    "particle": "tfc:item/metal/shield/bismuth_bronze_front"
+  },
+  "elements": [
+    {
+      "name": "h",
+      "from": [7, 10, 8],
+      "to": [8, 11, 13],
+      "faces": {
+        "north": {"uv": [0, 0, 0, 0], "texture": "#texture"},
```

Comment:
The `#missing` textures could probably be removed from the models entirely here.